apps/web/src/app/[locale]/(dashboard)/calendar/meet-together/page.tsx

```
@@ -109,7 +128,14 @@ export default async function MeetTogetherPage() {
   );
 }
 
-async function getData() {
+async function getData(
+  // wsId: string,
+  {
+    // q,
+    // page = '1',
+    // pageSize = '10',
+  }: { q?: string; page?: string; pageSize?: string }
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)